python/paddle/jit/layer.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+# Copyright (c) 2021 NVIDIA Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from paddle.fluid.core import Load
+
+
+class Layer():
+
+    def __init__(self):
+        self.cpp_layer = None
+        # {name: Function}
+        self.funciton_map = {}
```

Comment:
```suggestion
        self.functions = {}
```
有单词拼写错误，另外这里的dict需要使用WeakRef么？

Reply:
这里应该不需要WeakRef吧，funciton_map 持有的python端的 Function: `self.funciton_map[name] = Function(function)`。其余的意见已经修改，感谢